### PR TITLE
feat(words): add `fold_open` and `set_jump_point` config options

### PR DESCRIPTION
**Pull request by chrisgrieser**
## Description
Added `fold_open` and `set_jump_point` as config options for `Snacks.words.jump`.

### CODE REVIEW

File: docs/words.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ Auto-show LSP references and quickly navigate between them
   debounce = 200, -- time in ms to wait before updating
   notify_jump = false, -- show a notification when jumping
   notify_end = true, -- show a notification when reaching the end
+  fold_open = true, -- open folds after jumping
+  set_jump_point = true, -- set jump point before jumping
```
Comment (folke) [inline docs/words.md:17]: Can you rename this to `jumplist`?

File: docs/words.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ Auto-show LSP references and quickly navigate between them
   debounce = 200, -- time in ms to wait before updating
   notify_jump = false, -- show a notification when jumping
   notify_end = true, -- show a notification when reaching the end
+  fold_open = true, -- open folds after jumping
```
Comment (folke) [inline docs/words.md:16]: rename to `folds`?

**Review by folke — CHANGES_REQUESTED** [review]
Great! Just two renames :)

**Comment by chrisgrieser** [comment]
> Actually, maybe just remove the folds option and just always open folds.
I don't really see why someone would not have that option enabled...

Well, there is an option `:h foldopen`, so the behavior is configurable normally. Also, some people may prefer not to open all folds when they cycle the whole file

File: docs/words.md (hunk 2 above)
Comment (chrisgrieser) [reply docs/words.md:16]: sure about that? `fold_open` is quite verbose, while `folds` does not telegraph directly what the options does.

File: docs/words.md (hunk 1 above)
Comment (chrisgrieser) [reply docs/words.md]: sure

File: docs/words.md (hunk 2 above)
Comment (folke) [reply docs/words.md:16]: you're right. Maybe `foldopen` to resemble the option and cmd?

Comment (chrisgrieser) [reply docs/words.md]: 👍

**Comment by folke** [comment]
Awesome, thank you!

**Pull request merged**